extensions/schema-compare/src/test/models/schemaCompareOptionsModel.test.ts

```diff
@@ -0,0 +1,121 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import * as should from 'should';
+import * as mssql from '../../../../mssql/src/mssql';
+import {SchemaCompareOptionsModel} from '../../models/schemaCompareOptionsModel';
+
+describe('Schema Compare Options Model', () => {
+	it('Should create model and set options successfully', function (): void {
+		const model = new SchemaCompareOptionsModel(defaultOptions);
+		should.notEqual(model.getOptionsData(), undefined, "Options shouldn't be undefined");
+		should.notEqual(model.getObjectsData(), undefined, "Objects shouldn't be udnefined");
+
+		should.doesNotThrow(() => model.SetDeploymentOptions());
+		should.doesNotThrow(() => model.SetObjectTypeOptions());
+
+		should.equal(model.GetSchemaCompareOptionUtil(''), false, "Should return false if an invalid option is passed in");
```

Comment:
nit: I like the fluent way of typing better `should(model.GetSchemaCompareOptionUtil('')).be.false('Should return...');`